CAP/gap/CategoryMorphisms.gi

```diff
@@ -157,6 +157,12 @@ InstallMethod( AdditiveInverse,
                   
 AdditiveInverseForMorphisms );
 
+##
+InstallOtherMethod( \*,
```

Comment:
Please do not overload the operators. If you use a predefined data structure in CAP, it is unclear which function is called by the operator and may cause inconsistencies or errors.

Reply:
Please note that this was the issue where we discussed this back then: https://github.com/homalg-project/CAP_project/issues/139 .

We can leave the overloaded operator, but please be aware that it might (and almost certainly will) cause problems.